gaas-ui/test/setupTests.ts

```diff
@@ -1,5 +1,5 @@
 import Enzyme from "enzyme";
-import Adapter from "enzyme-adapter-react-16";
+import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
```

Comment:
Is this update unofficial?

Reply:
It looks to be an unofficial adaptor

Reply:
Have just had a quick google and found this:
https://stackoverflow.com/questions/64658031/which-enzyme-adapter-works-with-react-17

Reply:
Is this fine to use then? Wondering on React 16 vs 17?

Reply:
Your call on that one. The aim has been to upgrade the versions of different frameworks in order to minimise possible vulnerabilities. It seems as though this adaptor is the one which gets used when using react 17. Think 17 doesn't have a huge deal of changes over 16. The main one seems to be that in the future upgrading from 17 up is easier than in previous iterations.

Reply:
My opinion would be that its okay and it doesn't seem like enzyme is going to release an official adaptor anytime soon for react 17.